ferrocene/tools/generate-tarball/src/signatures.rs

```diff
@@ -97,14 +101,25 @@ fn hash_file(path: &Path) -> Result<Vec<u8>, Error> {
 
 #[cfg(test)]
 mod tests {
+    #[cfg(not(windows))]
```

Comment:
Why don't we just cfg the whole tests module?

Reply:
I was scared we'd add a test and not realize it's not running on Windows. 😅  I'll block cfg it.